[ADD][14.0] stock_unique_order_per_location + shopfloor_single_product_transfer_unique_order_at_location (#603)
OCA/wms: stock_unique_order_per_location/readme/DESCRIPTION.rst

This module adds an option on operation types.

> The `Only Same Order At Destination` option when checked, will only
> allow on a location products ordered for the same sales order.

Reviewer: ```suggestion
The `Do not mix sources at destination` option when checked, will only
allow on a location products coming from the same transfer.
```
